projects/go-tester/src/app/app.component.ts

```diff
@@ -81,4 +112,8 @@ export class AppComponent implements OnInit {
       }
     });
   }
+
+  toggleSideMenu(): void {
```

Comment:
Should we also move this code outside of the `app.component`?

Reply:
See above comment @devchris, but again, this is just for testing.